src/errors.ts

```diff
@@ -17,3 +18,16 @@ export function OAuth2Error(error?: OAuth2Errors): WebError {
 export class MongoUpdateError extends Error {}
 
 export class MongoInsertError extends Error {}
+
+export class MongoNoDataReturned extends Error {}
+
+export class GraphqlErrors extends Error {
+  parentErrors: string;
+
+  constructor(errors: readonly GraphQLError[]) {
+    super();
+    this.parentErrors = errors.map(({ message }) => message).join(" | ");
+  }
+}
```

Comment:
Here's a proposal to keep the idea of having only `parentError`

```suggestion
export class GraphqlError extends Error {
  parentError: GraphqlErrors;
  
  constructor(message, error: readonly GraphqlErrors) {
    super(message)
    this.parentError = error;
  }
}

export class GraphqlErrors extends Error {
  constructor(errors: readonly GraphQLError[]) {
    const message = errors.map(({ message }) => message).join("\n")
    super(message);
  }
}
```

and then, instead of 

```typescript
  if (result.errors) {
    throw result.errors;
  }
```

```typescript
  if (result.errors) {
    throw new GraphqlErrors(result.errors);
  }
```

That could be catched up the chain by a `new GraphqlError("graphql error", error)`

Reply:
I like it!